### DISCUSSION TITLE
Can update flag / option be implemented?

### DISCUSSION THREAD

**Discussion by necros2k7** [zhongfly/mpv-build#60] [Q&A] [answered]
Like in "yt-dlp -U" tool?  Downloading and updating binary to latest build

**Comment by zhongfly** [comment, marked as answer]
You can run `updater.bat` to get latest build if you use `mpv.exe`.
If you want a command like `mpv -u` that is impossible.([Same reason as ffmpeg](https://github.com/BtbN/FFmpeg-Builds/issues/249#issuecomment-1473719466))
If you really want that,please modify the source code of the mpv by yourself.
